dust_extinction/dust_extinction.py

```diff
@@ -8,7 +8,7 @@
 from astropy.modeling import (Model, Fittable1DModel,
                               Parameter, InputParameterError)
 
-__all__ = ['CCM89', 'FM90', 'P92', 'O94', 'F99',
+__all__ = ['CCM89', 'FM90', 'P92', 'O94', 'F99', 'F99FM07'
```

Comment:
There is a comma missing after 'F99FM07' in this line.  Causing errors with the build of the docs.